packages/manager/modules/telecom-dashboard/src/identity-check/form/cancellation/cancellation.controller.js

```diff
@@ -0,0 +1,44 @@
+export default class IdentityCheckFormCancellationCtrl {
+  constructor($translate, IdentityCheckService, TucToast) {
+    this.$translate = $translate;
+    this.IdentityCheckService = IdentityCheckService;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.trackPage('account-validation::cancel-validation-popup');
+  }
+
+  confirm() {
+    this.trackClick('cancel-validation-popup::confirm');
+
+    this.IdentityCheckService.cancelProcedure(this.procedureId)
```

Comment:
```suggestion
    return this.IdentityCheckService.cancelProcedure(this.procedureId)
```
And same for `this.goBack` calls, which returns promises